src/applications/static-pages/covid-vaccine-updates-cta/createCovidVaccineUpdatesWidget.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-export default function createCovidVaccineUpdatesWidget(store, widgetType) {
-  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
-  if (root) {
+export default function createCovidVaccineUpdatesWidget(store, _widgetType) {
+  const isCovidVaccineUpdatesPage =
+    document.location.pathname === '/health-care/covid-19-vaccine/';
+
+  if (!isCovidVaccineUpdatesPage) {
+    return;
+  }
+
+  const introText = document.querySelector('.va-introtext')?.nextElementSibling;
```

Comment:
Is there a note or documentation somewhere that says that this react widget is dependent on `.va-introtext` existing on the dom? 🙂

Reply:
Not right now...once the dust settles I can do that though